packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -209,6 +209,12 @@ export class SwirlFormControl {
 
     const isSelect = this.inputEl.tagName === "SWIRL-SELECT";
 
+    const tooltip: string | null = (() => {
```

Comment:
Question: Why the IIFE?

Reply:
I did not want to use a function which is called in the template so I thought this would be a good choice. Replaced it with:

`const tooltipEl = this.inputEl.getAttribute("tooltip");
const tooltip: string | null = tooltipEl !== null ? tooltipEl : null;`